web/js/contacts.react.js

```diff
@@ -14,6 +14,9 @@ var ContactSection = React.createClass({
         $.ajax({
             url: this.props.url,
             success: function (data) {
+                if (!data.contacts) {
+                    throw 'AJAX call for contacts appears to be corrupted! The response does not have a "contacts" key. Check the AJAX call!';
+                }
```

Comment:
This is not a "fix", but is probably something I should have added to my code originally. This makes it more obvious is there is a problem with the AJAX call, like if it is successful, but somehow we are getting an invalid response! In fact, in your case, this AJAX call was returning HTML, which ultimately caused the issue. Here's a screenshot of the AJAX call that should have been JSON:

<img width="640" alt="screen shot 2017-01-09 at 7 49 24 am" src="https://cloud.githubusercontent.com/assets/121003/21767223/bd584c9e-d640-11e6-860f-0ce38f8078ba.png">